blog/README.md

```diff
@@ -34,15 +33,16 @@ The main one, for the website as a whole, is located at [`docs/config/nav.yml`](
 
 Note here that each link assumes that the present working directory is `docs/docs/`, so for example the "Getting Started" README which is located in `docs/docs/getting-started/README.md` is listed as `getting-started/README.md`
```

Comment:
Do we also want to update "Getting Started" to "Tutotial" here? Also, maybe update the snippet in lines 25 - 32 to match the current `nav.yml`.